src/sagas/wallet.js

```diff
@@ -249,11 +262,8 @@ export function* startWallet(action) {
     reload: take(types.RELOAD_WALLET_REQUESTED),
   });
 
-  // We need to cancel threads on both reload and start
-  yield cancel(threads);
```

Comment:
No need to cancel manually, `takeLatest` will already take care of [cancelling all started sagas](https://redux-saga.js.org/docs/api/#takelatestpattern-saga-args)